### PR TITLE
Added PR template

### PR DESCRIPTION
**Pull request by Tais993**
[contributing]: https://github.com/Together-Java/TJ-Bot/wiki/Contributing
[code_guidelines]: https://github.com/Together-Java/TJ-Bot/wiki/Code-Guidelines
[new_issue]: https://github.com/Together-Java/TJ-Bot/issues/new/choose

## Pull-request

- [x] I have read the [contributing guidelines][contributing].
- [x] I have read the [code guidelines][contributing].
- [x] I have created a relating [issue][new_issue].

### Changes

- [ ] Internal code
- [X] New feature

Closes Issue: #131 

## Description
Adds template for new PR's

### CODE REVIEW

**Comment by Tais993** [comment]
Because I did

File: .github/PULL_REQUEST_TEMPLATE/feature_add.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[contributing]: https://github.com/Together-Java/TJ-Bot/wiki/Contributing
+[code_guidelines]: https://github.com/Together-Java/TJ-Bot/wiki/Code-Guidelines
+[new_issue]: https://github.com/Together-Java/TJ-Bot/issues/new/choose
+
+## Pull-request
+
+- [ ] I have read the [contributing guidelines][contributing].
+- [ ] I have read the [code guidelines][contributing].
```
Comment (marko-radosavljevic) [inline .github/PULL_REQUEST_TEMPLATE/feature_add.md:8]: These two link to the same thing.
```suggestion
- [ ] I have read the [contributing guidelines][contributing].
- [ ] I have read the [code guidelines][code_guidelines].
```

File: .github/PULL_REQUEST_TEMPLATE/feature_add.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- [ ] I have created a relating [issue][new_issue].
+
+### Changes
+
+- [ ] Internal code
```
Comment (marko-radosavljevic) [inline .github/PULL_REQUEST_TEMPLATE/feature_add.md:13]: Maybe:
```suggestion
- [ ] Existing code
```

Comment (Tais993) [reply .github/PULL_REQUEST_TEMPLATE/feature_add.md:13]: I didn't mean that but it should be that.

**Review by marko-radosavljevic — APPROVED** [review]

**Review by illuminator3 — APPROVED** [review]

**Comment by LovinLifee** [comment]
I'm extremely sorry, looks like I modified this by mistake. I'm not sure which checkboxes were originally checked.

**Comment by illuminator3** [comment]
> I'm extremely sorry, looks like I modified this by mistake. I'm not sure which checkboxes were originally checked.

fixed

**Pull request merged**